pallets/messages/src/tests/mock.rs

```diff
@@ -83,26 +84,34 @@ parameter_types! {
 	// should have this set large enough to accomodate the largest possible CID.
 	// Take care when adding new tests for on-chain (not IPFS) messages that the payload
 	// is not too big.
-	pub const MaxMessagePayloadSizeBytes: u32 = 73;
+	pub const MessagesMaxPayloadSizeBytes: u32 = 73;
 }
 
-impl std::fmt::Debug for MaxMessagePayloadSizeBytes {
+impl std::fmt::Debug for MessagesMaxPayloadSizeBytes {
 	fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
-		f.debug_struct("MaxMessagePayloadSizeBytes")
-			.field("v", &MaxMessagePayloadSizeBytes::get())
+		f.debug_struct("MessagesMaxPayloadSizeBytes")
+			.field("v", &MessagesMaxPayloadSizeBytes::get())
 			.finish()
 	}
 }
 
-impl PartialEq for MaxMessagePayloadSizeBytes {
+impl PartialEq for MessagesMaxPayloadSizeBytes {
 	fn eq(&self, _other: &Self) -> bool {
 		true
 	}
 }
 
-impl Clone for MaxMessagePayloadSizeBytes {
+impl Clone for MessagesMaxPayloadSizeBytes {
 	fn clone(&self) -> Self {
-		MaxMessagePayloadSizeBytes {}
+		MessagesMaxPayloadSizeBytes {}
+	}
+}
+
+impl Encode for MessagesMaxPayloadSizeBytes {}
+
+impl MaxEncodedLen for MessagesMaxPayloadSizeBytes {
+	fn max_encoded_len() -> usize {
+		MessagesMaxPayloadSizeBytes::get() as usize
```

Comment:
Is this right? Wouldn't the max encoded length of the constant be the max encoded length of a `usize` variable? ie, 4 or 8 bytes (or whatever the SCALE-encoded size of a `usize` is), not the value itself. Or am I misunderstanding here?

Reply:
Great catch!